requirements.txt

```diff
@@ -1,2 +1,2 @@
-Sphinx==1.8.2
-sphinx-bootstrap-theme==0.6.5
+Sphinx == 1.8.2
+(Sphinx - bootstrap - theme) == 0.6.5
```

Comment:
The change you are proposing, does not pass the tests
https://github.com/pgRouting/pgrouting/pull/1833/checks?check_run_id=1625921777#step:7:467
it fails to install the requirements.